docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -152,7 +152,10 @@ Available Options:
 ```bash title="Full Example"
 docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl upload \
 -H https://<username>:<accessKey>@api.us-west-1.saucelabs.com/api-testing/rest/v4/<hookId> \
--p /tests/test_abc
```

Comment:
@kimsauce why did you add "-v $(pwd)/tests:/tests " (lines 129 and 153)? keep it simple as the other examples